glue_jupyter/widgets/layer_options.py

```diff
@@ -72,6 +72,9 @@ def _update_layers_from_glue_state(*args):
         self.viewer.state.add_callback('layers', _update_layers_from_glue_state)
         _update_layers_from_glue_state()
```

Comment:
I don't fully understand why this isn't doing the trick (the add callback). Let me double check what is going on.